CHANGELOG.md

```diff
@@ -260,3 +261,4 @@ and may require changes in applications that invoke these methods:_
 [#153]: https://github.com/hamstar/Wikimate/pull/153
 [#155]: https://github.com/hamstar/Wikimate/pull/155
 [#157]: https://github.com/hamstar/Wikimate/pull/157
+[#161]: https://github.com/hamstar/Wikimate/pull/157
```

Comment:
Copy/paste not updated.

Reply:
Oh, wow. That got me thinking whether we should have a check for duplicate reference links as well! :see_no_evil:

Reply:
[Fixed](https://github.com/hamstar/Wikimate/compare/9c914fc45c0870d58cad75260959e07d79ee0e03..65089e79498117bc9e42173107572338e2c76947).